test/presubmit-tests.sh

```diff
@@ -73,6 +73,7 @@ function node_test() {
   local failed=0
   echo "Running node tests from $(pwd)"
   node_npm_install || failed=1
+  npm run bootstrap || failed=1
```

Comment:
bootstrap is already called in node_npm_install above

Reply:
Removing, was more for testing